app/styles/_virtual-machine-row.less

```diff
@@ -1,3 +1,17 @@
 .bar-separated + .bar-separated::before {
   content: ' | '
 }
+
+.vm-detail { }
+
+.vm-detail-key {
+  font-weight: 600;
+}
+
+.vm-detail-value {
+  padding-left: 0.25em;
```

Comment:
We typically use px values for padding, so it'd be good to be consistent here and below.

Reply:
done